velox/exec/tests/TableScanTest.cpp

```diff
@@ -4442,14 +4442,13 @@ TEST_F(TableScanTest, readFlatMapAsStruct) {
           makeFlatVector<int64_t>(kSize, folly::identity, nullEvery(5)),
           makeFlatVector<int64_t>(kSize, folly::identity, nullEvery(7)),
       })});
-  auto config = std::make_shared<dwrf::Config>();
-  config->set(dwrf::Config::FLATTEN_MAP, true);
-  config->set<const std::vector<uint32_t>>(dwrf::Config::MAP_FLAT_COLS, {0});
-  config->set<const std::vector<std::vector<std::string>>>(
-      dwrf::Config::MAP_FLAT_COLS_STRUCT_KEYS, {keys});
+  dwrf::WriterOptions options;
```

Comment:
Why are these dwrf options if we want to make this format agnostic?
If this is specifically targeting dwrf maybe we should use a different interface (writeToDwrfFile) and pass dwrf options. or we pass common options and let the common writer factory sort it out from the common options?
It seems a bit more clear to use a specific write function for the cases where it is needed to allow DrwfConfig to be used rather than trying to pollute the common options which can be used for more generic tests.

Reply:
I missed changing this to be `dwio::common::WriterOptions` type, thank you for catching it! Will change accordingly.